modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1328,6 +1305,36 @@ public void testAddSystemObjectDefinition() throws Exception {
 
 		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition);
 
+		// Enable localization
+
+		objectDefinition =
+			ObjectDefinitionLocalServiceUtil.addSystemObjectDefinition(
+				null, TestPropsValues.getUserId(), 0,
+				ObjectDefinitionTestUtil.getRandomName(), null, false, false,
+				true, true,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				false, "Test", null, null, null, null,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				false, ObjectDefinitionConstants.SCOPE_COMPANY, null, 1,
+				WorkflowConstants.STATUS_DRAFT,
+				Collections.singletonList(
+					new TextObjectFieldBuilder(
+					).labelMap(
+						LocalizedMapUtil.getLocalizedMap(
+							RandomTestUtil.randomString())
+					).name(
+						StringUtil.randomId()
+					).build()));
+
+		Assert.assertNotNull(
+			ObjectFieldLocalServiceUtil.getLocalizedObjectFields(
+				objectDefinition.getObjectDefinitionId()));
+
+		Assert.assertTrue(objectDefinition.isEnableLocalization());
+		Assert.assertTrue(objectDefinition.isUnmodifiableSystemObject());
+
+		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition);
```

Comment:
@rijema you can remove this, the other tests is enough. Thx!

Reply:
Removed.